index.html

```diff
@@ -2855,8 +3170,13 @@ <h2>
             <a>Queue a task</a> on the <a>user interaction task source</a> to
             run the following steps:
             <ol>
+              <li>Let <var>redactList</var> be the empty list. Optionally,
+              append the following items to <var>redactList</var>: «
+              "organization", "phone", "recipient", "addressLine" ».
```

Comment:
Do you have to append all of them together, or can you pick a subset? If all together, I'd say "optionally, set redactList to « ... »" If a subset, then I'd say "optionally add any of the following to redactList: ..." (the latter with no «»s)

In other places with optional steps, we include a nice note explaining why someone would do this. Can you do the same here?

Reply:
As the algorithm is specified, the `redactList` is always required (even if empty). Thus passing in the empty one should be ok. 

I did this because I didn't want to write "if `redactList` was passed and if includes 'xxx' ..." over and over. Hopefully that's ok 🤞